pagopa/io-app
# chore: [IOBP-951] Add payment verify generic error custom screen

## Short description
This PR adds a custom generic error screen when the payment verify fails;

## List of changes proposed in this pull request
- Added a new error `PaymentVerifyGenericErrorProblemJson` that has the fault code category equals to `PAYMENT_VERIFY_GENERIC_ERROR`;
- Force-mapped the `faultCodeDetail` when presenting the generic error from the verify screen page in order to send the right code to the assistance;

## How to test
- Start a payment from the `Payments` tab;
- After typing the notice code and the entity fiscal code, simulate an error response from the payment detail screen that should be showed after you type the entity fiscal code;

## Preview
|Before|After|
|-|-|
|<video src="https://github.com/user-attachments/assets/b4c7fe62-af24-49cc-b29b-b9853fc06daf" />|<video src="https://github.com/user-attachments/assets/becec9c1-3eaa-489f-945c-1d732ecd966e" />
